packages/server/modules/accessrequests/tests/streamAccessRequests.spec.ts

```diff
@@ -55,7 +55,7 @@ const cleanup = async () => {
 }
 
 describe('Stream access requests', () => {
-  let apollo: ApolloServer
+  let apollo: ServerAndContext
```

Comment:
Biggest annoyance - Apollo Server instance is no longer associated w/ the context builder, its now fed into the express middleware instead. When running operations against it manually, the context value has to be passed into the execute() call